### ISSUE TITLE
Ensure accessible keys aren't picked up by prompt auto-focus 

### ISSUE THREAD

**Issue by spiffyjr** [spiffyjr/spif-fe#2]
Some keys should be skipped from focusing prompt input since they're used for accessibility navigation.

* Arrow Keys
* Tabs

**Comment by spiffyjr** [comment]
Should be resolved by 647d2e583388edaa0c2e5a1ecf57336f68b294e0
